jena-arq/pom.xml

```diff
@@ -111,6 +111,18 @@
       <artifactId>commons-lang3</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.graalvm.js</groupId>
+      <artifactId>js</artifactId>
+      <scope>provided</scope>
```

Comment:
Actually, ARQ can be compiled on JDK < 15 *without* any Graal dependencies, and even on JDK 15 they can be mere *test* dependencies. Maybe it would rather make sense to include them as compile dependencies in Fuseki? @afs, what do you think?

Reply:
Making them `<scope>test</scope>` makes sense. At the moment, I think have the tests need this to test an optional feature is acceptable.

Including in Fuseki is another matter. scope=compile would mean the Jena project is shipping the GraalVM engine which itself is GPL2+classpath exception and not acceptable.

http://www.apache.org/legal/resolved.html#optional

While we can't ship it ourselves, we should provide instructions and scripts on how to add GraalVM.

There is a separate matter that GraalVM is quite large.

PMC email:
https://lists.apache.org/thread.html/raf8f65ee6451181560d36b98c6332677a751f86abf4abda2230d3240%40%3Cdev.jena.apache.org%3E

Reply:
I created another PR for that: https://github.com/apache/jena/pull/801